chat-widget/src/components/webchatcontainerstateful/common/DemoChatAdapter.ts

```diff
@@ -0,0 +1,87 @@
+import "rxjs/add/operator/share";
+import "rxjs/add/observable/of";
+import { Message } from "botframework-directlinejs";
+import { Observable } from "rxjs/Observable";
+import MockAdapter from "./mockadapter";
+import { customerUser, postBotMessageActivity, postBotAttachmentActivity, postBotTypingActivity, postEchoActivity, postSystemMessageActivity } from "./utils/chatAdapterUtils";
+import { createHeroCardAttachment, createJpgFileAttachment, createSigninCardAttachment, createThumbnailCardAttachment } from "./utils/attachmentActivityUtils";
+
+export class DemoChatAdapter extends MockAdapter {
+    constructor() {
+        super();
+
+        setTimeout(() => {
+            postSystemMessageActivity(this.activityObserver, "You're currently using a demo.", 0);

```

Comment:
are this messages going to be translated?

Reply:
No for the moment